### ISSUE TITLE
Version solving failed after updating to 1.0.0

### ISSUE THREAD

**Issue by dukaric1991** [mobxjs/mobx.dart#413]
After updating to 1.0.0 

```
  mobx_codegen: ^1.0.0
  flutter_mobx: ^1.0.0
  mobx: ^1.0.0
```

and when i run `flutter packages get` i get this error:

```
So, because app_flutter depends on mobx ^1.0.0, version solving failed.
pub get failed (1; So, because tennboard_flutter depends on mobx ^1.0.0, version solving failed.)
exit code 1
```

**Comment by nadeem-khawar** [comment]
Facing the same issue.

**Comment by pavanpodila** [comment]
Fix is in `1.0.1`. Please update once it shows up on pub.dev

**Comment by pavanpodila** [comment]
`flutter_mobx` and `mobx_codegen` is now at 1.0.1 and up on pub.dev. Please update.
